interface/interface.c

```diff
@@ -218,6 +270,154 @@ int process_resp_msg(void *buf)
 }
 #endif /* CP_BUILD */
 
+#if defined (CP_BUILD) && defined (MULTI_UPFS)
+void *dp_sock;
+void *dp_sock_context;
+void
+init_dp_sock(void)
+{
+	static char addr_string[128] = {0};
+	snprintf(addr_string, sizeof(addr_string),
+		 "%s://*:%u", "tcp", cp_nb_port);
```

Comment:
Any specific reason why we are not using cp_nb_addr also in the bind ?

Reply:
* represents `INADDR_ANY` over here.

Reply:
Yes understood. So control plane is binding on INADDR any and data plane is suppose to send the registration on the cp_nb_ip.  It looks good. But just that control plane is not binding to the specific address as per config. But looks good. I thought you missed it, but looks like you intentionally binding it to *.

Reply:
Yes.